library/core/src/fmt/num.rs

```diff
@@ -237,9 +245,9 @@ macro_rules! impl_Display {
         }
```

Comment:
Can't actually issue a "delete the code between here and here" suggestion, I guess, if it traverses "old" code.
```suggestion
```